packages/edit-site/src/components/sidebar-navigation-screen-global-styles/index.js

```diff
@@ -59,61 +60,109 @@ export function SidebarNavigationItemGlobalStyles( props ) {
 }
 
 function SidebarNavigationScreenGlobalStylesContent() {
-	const { storedSettings } = useSelect( ( select ) => {
-		const { getSettings } = unlock( select( editSiteStore ) );
+	const {
+		params: { path },
+	} = useLocation();
 
-		return {
-			storedSettings: getSettings(),
-		};
-	}, [] );
+	const variationsLink = useLink( {
+		path,
+		activeView: '/variations',
+	} );
+
+	const typographyLink = useLink( {
+		path,
+		activeView: '/typography',
+	} );
+
+	const colorsLink = useLink( {
+		path,
+		activeView: '/colors',
+	} );
+
+	const layoutLink = useLink( {
+		path,
+		activeView: '/layout',
+	} );
+
+	const blocksLink = useLink( {
+		path,
+		activeView: '/blocks',
+	} );
 
 	// Wrap in a BlockEditorProvider to ensure that the Iframe's dependencies are
 	// loaded. This is necessary because the Iframe component waits until
 	// the block editor store's `__internalIsInitialized` is true before
 	// rendering the iframe. Without this, the iframe previews will not render
 	// in mobile viewport sizes, where the editor canvas is hidden.
 	return (
-		<BlockEditorProvider
-			settings={ storedSettings }
-			onChange={ noop }
-			onInput={ noop }
-		>
-			<StyleVariationsContainer />
-		</BlockEditorProvider>
+		<>
+			<div className="edit-site-sidebar-navigation-screen-patterns__group-header">
+				<Heading level={ 2 }>{ __( 'Active' ) }</Heading>
+			</div>
+			<ItemGroup>
+				<SidebarNavigationItem
+					{ ...variationsLink }
+					aria-current={ false ? 'true' : undefined }
+				>
+					Moonlight
+				</SidebarNavigationItem>
+			</ItemGroup>
+			<div className="edit-site-sidebar-navigation-screen-patterns__group-header">
+				<Heading level={ 2 }>{ __( 'Customize' ) }</Heading>
+			</div>
+			<ItemGroup>
+				<SidebarNavigationItem
```

Comment:
Rather than duplicating this list, would it make sense to reuse this panel?
<img width="289" alt="Screenshot 2024-01-02 at 13 38 53" src="https://github.com/WordPress/gutenberg/assets/275961/2edc7bfc-9246-4f90-9a88-26d1c7a4e469">

Maybe not as these are quite different contexts?